docs/monitoring/inventory.md

```diff
@@ -1,13 +1,105 @@
 
-This should explain the new Inventory Monitoring feature, and have the following sections.
+The [Sematext Agent](../agents/sematext-agent) provides a simple and versatile way of gathering machine-related information such as host, VM, or container properties, kernel versions, and installed packages. It presents them on a per-host basis allowing you not only to view the data but also search and compare different hosts. All of this, in the same place, shipped automatically without any effort from an operations point of view.
 
-- overview
-- how it works
-- how to enable
-- what it does
-- what it solves 
-- how it helps with observability/troubleshooting/debugging/etc…
+The Inventory information is available in the Monitoring *Infrastructure* section of the *Monitoring* tab of your Sematext Cloud account - your main place for hosts, virtual machines and containers information. 
 
-Inventory monitoring collects info about packages installed on your infrastructure.  It also detects their (un)installation.
\ No newline at end of file
+![Sematext Inventory Main Screen](../images/monitoring/inventory_main_view.png)
+
+### Inventory Info
+
+The Inventory Info information provides a summarized view of your inventory information allowing you to see:
+
+- Number of different packages in your monitored environment,
+- Number of different OS distributions,
+- Number of different OS versions,
+- A number of different kernel versions.
+
+![Inventory Info](../images/monitoring/inventory_inventory_info.png)
+
+### Inventory Overview
+
+The Inventory overview provides a per-host view of each host, virtual machine, and container along with the operating system and its version, number of CPUs, their type, memory, number of hard disk drives and total capacity.
+
+![Inventory Overview](../images/monitoring/inventory_overview.png)
+
+Expanding each row gives detailed information about the system including:
+
+- Unique identifier of the system,
+- OS version,
+- Kernel version,
+- Java virtual machine version,
+- System type (bare metal, virtual machine, container),
+- Total memory size, 
+- Memory slot information,
+- Total disk size, 
+- Hard disk drives information.
+
+In addition, an overview of all the installed packages on a given system is available. This includes:
+
+- Installation source, 
+- Package name, 
+- Package version.
+
+![Inventory Overview](../images/monitoring/inventory_packages.png)
+
+### Comparing Hosts
+
+The Inventory UI allows comparing different systems to quickly and efficiently see differences between them:
+
+![Inventory Package Comparison](../images/monitoring/inventory_package_comparison.png)
+
+## How does it Work?
+
+The [Sematext Agent](../agents/sematext-agent) keeps track of all changes done on the file system and displaying them in the Inventory Monitoring UI when installing, upgrading or removing packages on the host system, in a virtual machine or container with:
```

Comment:
The [Sematext Agent](../agents/sematext-agent) keeps track of all changes done on the file system and displaying them in the Inventory Monitoring UI when installing, upgrading or removing packages on the host system, in a virtual machine or container with: => 
The [Sematext Agent](../agents/sematext-agent) keeps track of all changes done on the file system.  Such changes are displayed in the Inventory Monitoring UI.  Package installs, upgrades and removals are also captured. 
 They are displayed as Events.  The following types of packages are supported:

Reply:
Weird, I still see old info here.